examples/cellar/gen/http/storage/client/client.go

```diff
@@ -43,6 +49,10 @@ type Client struct {
 	decoder func(*http.Response) goahttp.Decoder
 }
 
+// StorageMultiAddEncoderFunc is the type to encode multipart request for the
+// "storage" service "multi_add" endpoint.
+type StorageMultiAddEncoderFunc func(*multipart.Writer, *[]*storage.Bottle) error
```

Comment:
Shouldn't the type used by `[]*storage.Bottle` and not `*[]*storage.Bottle`?

Reply:
Fixed